samples/keys/hw_unique_key/README.rst

```diff
@@ -0,0 +1,82 @@
+.. _hw_unique_key_usage:
+
+Hardware Unique Key
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows how to use a hardware unique keys (HUKs) to derive a key and use it for encryption, through the psa_crypto APIs.
+
+Overview
+********
+
+The sample does the following:
+
+* Initialize the HW if applicable
+  ** When TFM is enabled, it does this automatically on boot
+* Generate HUKs if applicable
+  ** When TFM is enabled, it does this automatically on boot
+* Generate a random IV
+* Derive a key
+  ** When TFM is enabled, this is done using the psa_crypto APIs
+  ** When TFM is not enabled, native APIs are used, and the key is imported into psa_crypto.
+* Encrypt a sample string using the key_id received from psa_crypto.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpuapp, nrf5340dk_nrf5340_cpuappns, nrf9160dk_nrf9160, nrf9160dk_nrf9160ns, nrf52840dk_nrf52840
```

Comment:
A few other samples are adding support for nRF21540. Should it be done here as well?

Reply:
No, this sample only works for a limited set of boards, and `nRF21540` is not one of them.